### PR TITLE
Remove the Tutorials, Samples, and Shared Resources section

### PR DESCRIPTION
**Pull request by Arhell**
issue https://github.com/kubeflow/website/issues/2759

### CODE REVIEW

**Comment by Arhell** [comment]
Kubeflow Samples - move to Getting Started / Examples & rename
Codelabs, Workshops, and Tutorials - remove
Blog Posts - remove
Videos - remove
Shared Resources and Components - remove

/assign @RFMVasconcelos

**Comment by Arhell** [comment]
/assign @thesuperzapper
https://github.com/kubeflow/website/pull/2846#issuecomment-889497926

**Comment by Arhell** [comment]
cc @RFMVasconcelos

**Comment by Arhell** [comment]
сс @RFMVasconcelos

**Comment by thesuperzapper** [comment]
Thanks @Arhell, looks good!

@RFMVasconcelos can you please review/approve?

/lgtm

**Review by RFMVasconcelos — COMMENTED** [review]
Many thanks for all these contributions @Arhell ! You rock! :rocket: 

Would it be possible to add a redirect to `redirects` file? As we're moving things around, so that external links don't get broken :)

**Comment by Arhell** [comment]
> Many thanks for all these contributions @Arhell ! You rock! 🚀
> 
> Would it be possible to add a redirect to `redirects` file? As we're moving things around, so that external links don't get broken :)

Done

File: content/en/_redirects [hunk 1]
```diff
@@ -96,6 +96,9 @@ docs/started/requirements/                    /docs/started/getting-started/
 # Remove Kubeflow installation on existing EKS cluster
 /docs/aws/deploy/existing-cluster/             /docs/distributions/aws/deploy/install-kubeflow/
 
+# Remove examples docs
+/docs/examples/*                                /docs
```
Comment (RFMVasconcelos) [inline content/en/_redirects:100]: As we are moving things from `/docs/examples` to `/docs/started/kubeflow-examples` could you add the latter in this line? 

Thank you for this!!

Comment (Arhell) [reply content/en/_redirects:100]: Ok

**Comment by Arhell** [comment]
@RFMVasconcelos  done

**Review by RFMVasconcelos — COMMENTED** [review]
Thank you @Arhell !! 

/lgtm
/approve

(you will need to remove draft status for merge)

**Pull request merged**